docs/development/sync-protocol.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Santa Sync Server Protocol
+parent: Development
+mermaid: true
+---
+
+{% if page.mermaid %}
```

Comment:
These lines appear to be rendered as text in Github, is that intentional?

Reply:
It's not. This was a workaround for the diagrams to appear on santa.dev. Which I think is the ultimate destination for these docs.